src/raiden_libs/matrix.py

```diff
@@ -145,15 +145,23 @@ def __init__(
         self.service_room_suffix = service_room_suffix
         self.message_received_callback = message_received_callback
 
-        if servers:
-            self.available_servers = servers
-        else:
-            self.available_servers = get_matrix_servers(
+        try:
+            known_servers = get_matrix_servers(
                 DEFAULT_MATRIX_KNOWN_SERVERS[Environment.PRODUCTION]
                 if chain_id == 1
                 else DEFAULT_MATRIX_KNOWN_SERVERS[Environment.DEVELOPMENT]
             )
 
+        except RuntimeError as ex:
+            if servers is None:
+                raise RuntimeError from ex
```

Comment:
When you're not adding anything to an exception you can simply `raise` without any arguments that way the original exception will be re-raised.